### ISSUE TITLE
Something appears to have broken Indigo CSL-M styles

### ISSUE THREAD

**Issue by JCBerger** [Juris-M/style-modules#279]
I think something from the December 2020 updates might have broken the Indigo CSL-M styles on the Juris-M website ([https://juris-m.github.io/styles/](https://juris-m.github.io/styles/)). I keep receiving the error "Oops. Novel citation pattern" for court cases — for both the "Book" and the "Law Review" styles.

**Comment by JCBerger** [comment]
Could this be the result of the CSL file referring to a "case" as a "legal_case" while the dropdown menu in JurisM simply calls references "cases"?

**Comment by bwiernik** [comment]
> Could this be the result of the CSL file referring to a "case" as a "legal_case" while the dropdown menu in JurisM simply calls references "cases"?

No, this is just a localization of the CSL item type label into various languages.

**Comment by bwiernik** [comment]
Can you describe exactly what you are doing and what error you are getting? When exactly is this happening?

**Comment by fbennett** [comment]
This can happen when the Jurism record has an unexpected pattern of data.
Two things will help us get it fixed for you:

* Please export a mis-cited item with (in the center pane) right-click,
Export item, and selecting CLS-JSON as the export format, then either post
the file here as an attachment, or paste the content into a message.

* Also, let us know the source of the failing items. That will help us be
sure that things work end-to-end without error to the extent possible.

Frank

**Comment by PolkaDotKat** [comment]
I was having the same issue. I had a heading with a colon in the "extra" field and when I removed that it seemed to solve the problem. Strangely, this wasn't causing issues a few days ago, so I'm not fully sure what caused it. Hopefully that helps someone else!

**Comment by bwiernik** [comment]
What was the heading?

**Comment by PolkaDotKat** [comment]
"History:"
